packages/file/src/touch.coffee.md

```diff
@@ -31,24 +31,23 @@ require('nikita')
     on_action = ({config, metadata}) ->
       # Options
       config.target = metadata.argument if metadata.argument?
-      throw Error "Required \"target\" option" unless config.target
 
 ## Schema
 
     schema =
       type: 'object'
       properties:
         'gid':
-          $ref: 'module://@nikitajs/file/src/index#/properties/gid'
+          $ref: 'module://@nikitajs/engine/src/actions/fs/base/chown#/properties/gid'
         'mode':
-          $ref: 'module://@nikitajs/file/src/index#/properties/mode'
+          $ref: 'module://@nikitajs/engine/src/actions/fs/base/chmod#/properties/mode'
         'target':
           oneOf: [{type: 'string'}, {typeof: 'function'}]
           description: """
-          File path where to write file.
+          File path where to write file or a callback which will be transformed to a valid file path.
```

Comment:
It is not a callback (they are for other purpose), just a function, it doesn't really transform (at least we don't care what's the logic inside), the contract is that it must return a valid path.